site/templates/collection/carousel/item/synopsis.jet

```diff
@@ -0,0 +1,5 @@
+{{block carouselItemSynopsis(synopsis)}}
```

Comment:
I think we should change this code and filename away from "synopsis". In Admin "Synopsis" is a different field while this field is "Short Description" (and the API is "tagline" - confusing I know). There's no right answer here, but I'm leaning towards "description" with "tagline" being a bit overloaded.

Reply:
Just rolling with the existing class names.  Ideally everything should match up with names used in the datasource.